chicken_app/src/Pages/Signup.js

```diff
@@ -31,7 +31,7 @@ function Signup(props) {
   const [authError, setAuthError] = useState({ isAuthError:false, errorDetails:""})
   const [errors, setErrors] = useState(
     { 
-      emailErrorText:"", isEmailValid: true, 
+      emailErrorText:"email needs to include @ symbol", isEmailValid: true, 
```

Comment:
This error shows as soon as you open the page, and also after you add the @ symbol in the email.

Reply:
Don't think we should display this. We should maintain consistsncy with what other websites do. cases of:
Email is invalid
email already in use(we'll need to check for this err in the future)

That are checked when you hit submit
are good enough

Reply:
I think Yellow means that you should swap the message with 'Email is invalid'

and if the email is in use say 'Email is already in use'